Tau sprites & minimal code, Imperial Guard Halfmasks, Icon improvements.

FantasticFwoosh opened the pull request:
## Tau Additions & Tweaks
- Adds tau-guevesa (human-helper) helmets & a retouched fire warrior helmet, with extra large obj icons.

- Retouches tau fire warrior armor to be more detailed, especially around the legs & shins.

- Sprites & very lightly codes to implement tau-radio headsets, cypher keys are sprited but not implemented,

## Other Changes

- Equalizes a small assortment of missing suit obj assets, detailed in my first commit. More needs to be added before ``obj/clothing/suits.dmi`` and ``mob/onmob/suit.dmi`` are level and can support each other in the same capacity.

- Fixes lord-commander cape & ``commisar 3`` icons that were commited incorrectly to ``obj/clothing/suits.dmi`` with their mob sprites.

- Adds a IG halfmask which looks more like the in-universe equivilent, seperate as a subtype of the halfmask in code.

Valterak commented:
noticed that the armor kill thing got added back in, probably on accident, so I am now killing it again.

FantasticFwoosh committed:
adds fire warrior & goose station assets

Adds a fire-warrior suit icon, ads inq armor assets (``inqarmor`` , ``inqarmor_hooded,  ``inqcoat``) , ``warlock``,  ``skitrobe``, ``acolytecoat`` from cadiastation repo.

Fixes ``lccoat`` and ``commisar3`` incorrect mob sprites ported from cadiastation repo

FantasticFwoosh committed:
add fire warrior suit asset

``fw_armor``

FantasticFwoosh committed:
Adds IG halfmask assets

``ig_halfgas``

FantasticFwoosh committed:
Adds Tau head assets.

``fw_helmet`` and ``guevesa``

FantasticFwoosh committed:
Adds tau headset assets

``guevesa_tactical_headset`` to fit on humans, ``tau_tactical_headset`` to fit on tau racial members,``fw_headset``, ``tau_headset``

FantasticFwoosh committed:
Adds tau radio assets

``t_headset_f``, ``t_tactical_headset_f``, ``t_headset`` and two cypher keys: ``tau_cypherkey``, ``tau_cypherkey_f``

FantasticFwoosh committed:
Codes in imperial guardsmen halfmask

functionally similar, but stylized in-universe. Haven't tested to see if it lowers.

FantasticFwoosh committed:
Rudimentary tau headsets

Not currently connected to a network via cypher key. Includes commented recommendation to add code switching between racial model proportions (based on cadiastation tau sprites)
